ISHHoverBar/ISHHoverBar.m

```diff
@@ -170,6 +170,11 @@ @interface ISHHoverBar ()
 @property (nonatomic, nonnull) ISHHoverSeparatorView *separatorView;
 @property (nonatomic, nonnull) ISHHoverShadowLayer *shadowLayer;
 @property (nonatomic, nullable) NSArray<UIControl *> *controls;
+
+/// Maintains a lookup table of UUIDs (same as used in itemsDictionary) for controls
+@property (nonatomic, nullable) NSDictionary<NSString *, UIControl *> *controlsDictionary;
```

Comment:
Good point. Even better we can use weakToWeakObjectsMapTable with `NSMapTable<UIControl *, UIBarButtonItem *> *itemsControlsMap`.